tests/app/celery/test_lookup_va_profile_id_task.py

```diff
@@ -269,3 +269,48 @@ def test_caught_exceptions_should_set_status_reason_on_notification(
     mocked_update_notification_status_by_id.assert_called_with(
         notification.id, notification_status, status_reason=failure_reason
     )
+
+
+@pytest.mark.parametrize(
+    "exception, reason",
+    [
+        (BeneficiaryDeceasedException('some error'), BeneficiaryDeceasedException.failure_reason),
+        (IdentifierNotFound('some error'), IdentifierNotFound.failure_reason),
+        (MultipleActiveVaProfileIdsException('some error'), MultipleActiveVaProfileIdsException.failure_reason)
```

Comment:
There are 6 exceptions that will execute the code you are testing, but you've only parametrized 3 of them.  Is there a reason you omitted the other 3?

Reply:
Sorry about not covering all the test cases, I totally forgot. (Maybe I was too excited about fixing the problem we had). I will write the additional tests today and test them out.